GPU/Common/FramebufferCommon.h

```diff
@@ -379,6 +379,7 @@ class FramebufferManagerCommon {
 	bool usePostShader_ = false;
 	bool postShaderAtOutputResolution_ = false;
 	bool postShaderIsUpscalingFilter_ = false;
+	bool postShaderIsSSAAFilter_ = false;
```

Comment:
Why not make this an integer, so you can make an even more extreme 3x3 SSAA filter in the future? :)

Reply:
:3 done.